README.md

```diff
@@ -1,5 +1,12 @@
-## Cortex Certifai Examples 
-This directory contains example Jupyter notebooks illustrating how to use Certifai in a notebook environment.
+# Cortex Certifai Examples 
+This directory contains examples and tutorials illustrating how to use Certifai in a notebook environment and CLI.
```

Comment:
Change to: This directory contains example files, notebooks, and tutorials that will help you get started using Certifai tools.